prepare_offline_installation/prepare_offline_installation.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# simplify your orcharhino Server offline installation by generating commands based on a list of repositories
+
+###################################################################
+# adapt path to your offline medium, for example an external disk
+repo_path = "/media/my_disk/my_repos/"
+# adapt list of repositories
+repositories = [
+    "Insert list from ATIX Service Portal:",
+    "https://atixservice.zendesk.com/hc/de/articles/4414413251346",
+]
+###################################################################
+
+# create repo file
+repo_file = "/tmp/orcharhino.repo"
+repo_file_content = """"""
```

Comment:
Two double-quotes are enough to signal an empty string :grin: 
Also applies to the two other definitions using this to define an empty string.

Reply:
Applied.